app/config/index.js

```diff
@@ -230,6 +230,36 @@ const config = convict({
     default: '',
     env: 'METABASE_SITE_URL',
   },
+  SP_SITE: {
```

Comment:
Worth updating the `.env.example` file with these? Since the endpoint will fail without them

Reply:
Agreed, adding a sample.